gr-dtv/lib/dvbt2/dvbt2_paprtr_cc_impl.cc

```diff
@@ -751,8 +752,8 @@ namespace gr {
               papr_fft->execute();
               memcpy(ones_time, papr_fft->get_outbuf(), sizeof(gr_complex) * papr_fft_size);
               volk_32fc_s32fc_multiply_32fc(ones_time, ones_time, normalization, papr_fft_size);
-              memset(&r[0], 0, sizeof(gr_complex) * N_TR);
-              memset(&c[0], 0, sizeof(gr_complex) * papr_fft_size);
+              fill_n(&r[0], N_TR, 0);
```

Comment:
Are you `using namespace std;` or why does this work without the `std::`?

Reply:
Thanx for catching that. I definitely meant to use std::. I must of checked it 5 times, but still overlooked it.

As to why it still compiles, I'm not sure. I did notice that "using namespace std;" is included in all xxx_swigPYTHON_wrap.cxx swig files. For example, build/gr-dtv/swig/dtv_swigPYTHON_wrap.cxx

How it finds the function is also interesting.
```
In file included from /usr/include/c++/8/memory:62,
                 from /usr/include/boost/config/no_tr1/memory.hpp:21,
                 from /usr/include/boost/smart_ptr/shared_ptr.hpp:23,
                 from /usr/include/boost/shared_ptr.hpp:17,
                 from /home/re/xfer/gnuradio/gnuradio-runtime/include/gnuradio/sptr_magic.h:26,
                 from /home/re/xfer/gnuradio/gnuradio-runtime/include/gnuradio/basic_block.h:27,
                 from /home/re/xfer/gnuradio/gnuradio-runtime/include/gnuradio/block.h:28,
                 from /home/re/xfer/gnuradio/gnuradio-runtime/include/gnuradio/sync_block.h:27,
                 from /home/re/xfer/gnuradio/gr-dtv/include/gnuradio/dtv/dvbt2_paprtr_cc.h:27,
                 from /home/re/xfer/gnuradio/gr-dtv/lib/dvbt2/dvbt2_paprtr_cc_impl.h:24,
                 from /home/re/xfer/gnuradio/gr-dtv/lib/dvbt2/dvbt2_paprtr_cc_impl.cc:25:
/usr/include/c++/8/bits/stl_algobase.h:
```

I pushed a squashed commit with the fix.